server/config-example.js

```diff
@@ -34,4 +34,4 @@ config.couchDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchDbSer
 config.couchAuthDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchCredentials() + config.couchDbServer + ':' + config.couchDbPort;
 // config.searchURL = 'http://localhost:9200'; ELASTIC SEARCH URL (OPTIONAL)
 config.serverInfo = 'Development Ember CLI server';
-module.exports = config;
+module.exports = config;
```

Comment:
@donaldwasserman correct... _executeContainsSearch is what ends up calling to /search/hrdb/ which is a proxy to ElasticSearch (or a temp view search using couchdb).  

This part of the code needs to have the size parameter passed to it:
```javascript
Ember.$.ajax(searchUrl, {
  dataType: 'json',
  data: {
    q: queryString,
    size: variableWithSize //Replace variableWithSize with whatever variable has the size.
  },
  success: successFn
});
```

Also, even if you don't have ElasticSearch installed you can still test to see if the value is being passed along by running a search from Patient and looking in the developer console to see what url the app is passing along to the search proxy.  The url should end with ```&size=25```